pkg/plugins/kafka/source.go

```diff
@@ -95,16 +93,21 @@ func (s *Source) startFrom(position record.Position) error {
 	return nil
 }
 
-func toRecord(message *kafka.Message, position string) (record.Record, error) {
-	return record.Record{
+func toRecord(message *kafka.Message, position string) (sdk.Record, error) {
+	return sdk.Record{
 		Position:  []byte(position),
 		CreatedAt: time.Time{},
```

Comment:
> `CreatedAt` represents the time when the change occurred in the source system. If that's impossible to find out, then it should be the time the change was detected by the connector.

Not sure if `*kafka.Message` contains a timestamp of when it was created, otherwise we should use `time.Now()`.

Reply:
Good point 👍 Done.